### ISSUE TITLE
Tab Should Not Activate on Down Click

### ISSUE THREAD

**Issue by shavonne-sage** [Sage/carbon#4640]
### Desired behaviour

When I use my mouse to activate a tab, it is activated on up-click, thus meeting WCAG's [Success Criterion 2.5.2 Pointer Cancellation (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html).

### Current behaviour

When I use my mouse to activate a tab, it is activated on down-click.

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by shavonne-sage** [comment]
I created this as a feature request because I was unable to save it as a bug.

Please consult @tempertemper if you have any questions regarding the accessibility criteria. Thank you.

**Comment by edleeks87** [comment]
FE-4509 the fix should be done as part of this ticket
